### PR TITLE
Add CIS benchmark rules 1.1.20 & 1.1.21

### PR DESCRIPTION
**Pull request by uri-weisman**

### CODE REVIEW

File: compliance/cis_k8s/rules/cis_1_1_20/rule.rego [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package compliance.cis_k8s.rules.cis_1_1_20
+
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+# Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Manual)
+finding = result {
+	common.file_match_pattern("/etc/kubernetes/pki/[a-zA-Z]+.crt", data_adapter.file_path)
```
Comment (oren-zohar) [inline compliance/cis_k8s/rules/cis_1_1_20/rule.rego:8]: Can't a CRT file name contain numbers? Also, I'm not a fan of regex here, can we avoid it? Why not validate the path and the filename separately?

File: compliance/cis_k8s/rules/cis_1_1_20/rule.rego [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package compliance.cis_k8s.rules.cis_1_1_20
+
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+# Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Manual)
+finding = result {
```
Comment (oren-zohar) [inline compliance/cis_k8s/rules/cis_1_1_20/rule.rego:7]: nit: please separate to three parts to make it easier to read: `# filter` - `# evaluate` - `# set result`

File: compliance/cis_k8s/rules/cis_1_1_20/rule.rego (hunk 1 above)
Comment (uri-weisman) [reply compliance/cis_k8s/rules/cis_1_1_20/rule.rego:8]: Sure, we can do it in two steps..

File: compliance/cis_k8s/rules/cis_1_1_21/rule.rego [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package compliance.cis_k8s.rules.cis_1_1_21
+
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+# Ensure that the Kubernetes PKI key file permissions are set to 600 (Manual)
+finding = result {
+	# filter
+	common.file_in_path("/etc/kubernetes/pki", data_adapter.file_path)
+	contains(data_adapter.filename, ".key")
```
Comment (oren-zohar) [inline compliance/cis_k8s/rules/cis_1_1_21/rule.rego:10]: super nit: 
```suggestion
	endswith(data_adapter.filename, ".key")
```

**Review by oren-zohar — APPROVED** [review]
looks good,  I would use `endswith` instead but not a must, approved.

**Pull request merged**